Tests/BasicsTests/NetrcTests.swift

```diff
@@ -13,8 +13,6 @@
 import Basics
 import XCTest
 
-/// Netrc feature depends upon `NSTextCheckingResult.range(withName name: String) -> NSRange`,
-/// which is only available in macOS 10.13+ at this time.
```

Comment:
This comment seems like something that was here explaining an availability or `#if` statement, but that doesn't exist here anymore.